src/System.Net.Ping/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net.NetworkInformation.Tests
     public class LoggingTest
     {
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot, "NetEventSource is only part of .NET Core and requires internal Reflection")]
```

Comment:
Most of the NetEventSource related tests are disabled across the board for all networking libraries when run in UWP contexts.  We need to investigate how to make the logging work in UWP context (future work item).